core/domain/feedback_domain.py

```diff
@@ -53,15 +83,46 @@ def to_dict(self):
 
     @staticmethod
     def get_exp_id_from_full_thread_id(full_thread_id):
+        """Returns the ID of the exploration that the thread belongs to.
+        Args:
+            full_thread_id: str. The ID of the feedback thread.
```

Comment:
`The ID of the feedback thread.` --> `The "full ID" of the feedback thread. (This is a concatenation of the exploration ID and the thread ID.)`